CONTRIBUTING.rst

```diff
@@ -135,7 +135,7 @@ Before you submit a pull request, check that it meets these guidelines:
    Travis-CI.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
-   feature to the list in README.md.
+   feature to the list in CHANGELOG.rst.
```

Comment:
sorry i think i made an unclear comment. i think README.md was correct here but if am not mistaken the travis link two lines below is not what we use anymore